scripts/utilities/update_revision.sh

```diff
@@ -78,13 +84,30 @@ source ${1};
 
 # If ATSRC_PACKAGE_CO is not defined, we skip the revision check.
 if ! [ -n "${ATSRC_PACKAGE_CO}" ] || ! [ -n "${ATSRC_PACKAGE_REV}" ]; then
-	return 0;
+	exit 0;
 fi
 
-# This function print a message with identation
+if [[ ${service} = "github" ]];
```

Comment:
Be careful with unquoted comparisons.
They can fail if `${service}` is undefined, a null string or a string whitespace string, e.g. `"  "`.
Even if the current code tests it earlier, belt-and-suspenders requires us to quote it.